### ISSUE TITLE
ObjectLoadError has a weird string

### ISSUE THREAD

**Issue by Nubbie** [OpenRCT2/OpenRCT2#6663]
**OS:** Win10
**Version:** 0.1.2
**Commit/Build:** [e.g. 426e106]

The window that warns about missing stuff seems to be a bit buggy in plain English sometimes, but changing language really mess with it

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Multiplayer?

**Steps to reproduce:**
1. Load a save with missing assets 
2. Change language

**Screenshots / Video:**
![image](https://user-images.githubusercontent.com/17282384/32747843-66ffb3f6-c8ba-11e7-8778-cdbd8d82634b.png)

**Save game:**

[Missing Assets.txt](https://github.com/OpenRCT2/OpenRCT2/files/1468388/Missing.Assets.txt)

**Comment by Nubbie** [comment]
It seems like it takes strings from objects, not directly from the language file, as the string highlighted ins't in sv-SE

**Comment by rwjuk** [comment]
o_o

**Comment by marijnvdwerf** [comment]
My guess is that it's some sort of pointer deallocation issue in the WindowManager that calls `window_object_load_error_open`

**Comment by rwjuk** [comment]
@marijnvdwerf Good hypothesis - testing now either way!
